docroot/themes/contrib/civictheme/config/install/civictheme.settings.yml

```diff
@@ -10,6 +10,7 @@ civictheme_site_logo_alt: 'CivicTheme logo'
 civictheme_header_theme: light
 civictheme_footer_theme: dark
 civictheme_footer_background_image: ''
+civictheme_override_domains: ''
```

Comment:
please refactor this to be an array

```
      label: 'List of override domains'
      type: sequence
      sequence:
        type: string
```

Reply:
Not sure, The configurations mentioned are fro schema. 
settings.yml just stores the default actual value.